spec/mailers/notify_mailer_spec.rb

```diff
@@ -27,6 +27,22 @@
     allow(ENV).to receive(:fetch).with('NOTIFY_RESET_PASSWORD_TEMPLATE_ID').and_return('reset-password-template')
     allow(ENV).to receive(:fetch).with('NOTIFY_CHANGE_PASSWORD_TEMPLATE_ID').and_return('change-password-template')
     allow(ENV).to receive(:fetch).with('NOTIFY_NEW_CASE_SAVED_TEMPLATE_ID').and_return('new-case-saved-template')
+    allow(ENV).to receive(:fetch).with('NOTIFY_STATISTICS_REPORT_TEMPLATE_ID').and_return('statistics-report')
+  end
+
+  describe '#statistics_report' do
+    before do
+      allow(ENV).to receive(:fetch).with('STATISTICS_REPORT_EMAIL_ADDRESS').and_return('reporting@tax-tribunals.org')
```

Comment:
That's a good point, but a) I don't like putting expectations in a before block, and b) that wouldn't work, because not all the specs fetch all the environment variables.
So, I've left is as 'allow' in the before block, but added explicit tests that we really are fetching values from ENV.